src/ancillary/codecs.ts

```diff
@@ -1,39 +1,35 @@
-import * as t from 'io-ts'
 import { CID } from 'multiformats/cid'
 import { CommitID, StreamID } from '@ceramicnetwork/streamid'
 import * as uint8arrays from 'uint8arrays'
 import { isDIDString } from './did-string.js'
+import * as t from 'codeco'
```

Comment:
Rather than a `*` import, can we just bring in `TrivialCodec` and `IContext`? If we do need a `*` import, rename to something like `cdc` or `codec`.

Reply:
Oh, here it is! The suggestion! Accepted.